checker-plugin/src/test/scala/org/virtuslab/akkasaferserializer/PluginComponentSpec.scala

```diff
@@ -55,4 +55,9 @@ class PluginComponentSpec extends AnyFlatSpecLike with should.Matchers {
   it should "be able to detect serializer trait in generics" in {
     testCode(getResourceAsString("GenericsTest.scala"))
   }
+
+  it should "detect lack of upper bounds in generics" in {
```

Comment:
Does this PR fix both `akka-safer-serializer-failing` and `akka-safer-serializer-exception` branches from Hydra repo? I'm not sure if both were corresponding to the same error or two different ones 🤔

Reply:
`sbt compile` reported error on `akka-safer-serializer-failing` but didn't report anything on `akka-safer-serializer-exception`.
Maybe `akka-safer-serializer-exception`  was about some false positive that was already solved?